Userland/Libraries/LibTextCodec/indexes.json

```diff
@@ -0,0 +1,36 @@
+{
```

Comment:
This sure is one of the tables of all time.

Does it have a canonical (versioned) spec location that we could download it from instead of committing it to our repo?

Reply:
The source for this is https://encoding.spec.whatwg.org/indexes.json
It's not versioned and I didn't really want to deal with `ENABLE_NETWORK_DOWNLOADS` and friends.

Reply:
Hmm all right. We should at least mention the source somewhere in the changes (in the code generator? the cmake lists?) so that it's clearer where the data came from and where to look for updates when we need to make them.

Reply:
Good idea. Added the URL above the `invoke_generator()` call